tracking/src/va.js

```diff
@@ -0,0 +1,27 @@
+const SessionManager = require('./SessionManager.js');
+const DataCollection = require('./DataCollection.js');
+const Tracker = require("./Tracker.js");
+
+
+module.exports = {
+    
+
+    init: function(obj) {
+        let code = obj.code; 
+        let uid = obj.uid; 
+        let country = obj.country; 
```

Comment:
should change this to boolean as mention above

Reply:
changed